tabcmd/commands/auth/session.py

```diff
@@ -374,12 +380,13 @@ def _read_from_json(self):
                 self.no_proxy = auth["no_proxy"]
                 self.proxy = auth["proxy"]
                 self.timeout = auth["timeout"]
-        except KeyError as e:
-            self.logger.debug(_("sessionoptions.errors.bad_password_file"), e)
```

Comment:
Is there any harm in going from a translated string to a non-translated string? Also, now that the `bad_password_file` string is no longer referenced, is there a process to delete it?

Reply:
The string is referenced elsewhere, it was being re-used in a bit of an awkward way here.  Not making it a translated string is a little lazy, but I have another CR in progress to figure out the best way to add new ones to the translation files.